### DISCUSSION TITLE
[2.1.864] New model list

### DISCUSSION THREAD

**Discussion by lllyasviel** [lllyasviel/Fooocus#2067] [Announcements]
Below model list will be used by Fooocus since 2.1.864 as default:

| Task | Windows | Linux args | Main Model | Refiner | Config                                                                         |
| --- | --- | --- | --- | --- |--------------------------------------------------------------------------------|
| General | run.bat |  | juggernautXL_v8Rundiffusion | not used | [here](https://github.com/lllyasviel/Fooocus/blob/main/presets/default.json)   |
| Realistic | run_realistic.bat | --preset realistic | realisticStockPhoto_v20 | not used | [here](https://github.com/lllyasviel/Fooocus/blob/main/presets/realistic.json) |
| Anime | run_anime.bat | --preset anime | animaPencilXL_v100 | not used | [here](https://github.com/lllyasviel/Fooocus/blob/main/presets/anime.json)     |

If you install Fooocus after 2024 Jan 27, you do not need to do anything and all downloads are automatic.

To make the software user-friendly, if you already have previous versions of JuggernautXL or realisticStockPhoto, Fooocus will not download new models.

To always download new models, use `--always-download-new-model`

The JuggernautXL and RealisticStockPhoto are just version difference and they should be similar to before

The anime model is animaPencilXL_v100. Note that:

animaPencilXL_v100 = [animagine-xl-3.0](https://huggingface.co/cagliostrolab/animagine-xl-3.0) + blue_pencil-XL

blue_pencil-XL = [a merge of about 150 sdxl models](https://huggingface.co/bluepen5805/blue_pencil-XL) (including [Kohaku-XL beta](https://civitai.com/models/162577/))

In the final round of test, the model animaPencilXL_v100 and [AAMXL(another animagine v3 mix)](https://civitai.com/models/269232?modelVersionId=303526) are both slightly more flexiable to use than animagine-xl-3.0, especially when the prompt is short or casually written. (the original animagine-xl would require formatted prompts, and shorter prompts or casual prompts sometimes will cause artifacts or saturation problems. Using a merge from animagine-xl, like animaPencilXL_v100 or AAMXL, can fix this problem.)

# Sanity Check

**Anime Preset**, Default outputs with same seed 12345

![image](https://github.com/lllyasviel/Fooocus/assets/19834515/e9be2fff-a86f-47a6-9922-83460a979fc1)

### 1girl, garden

![image](https://github.com/lllyasviel/Fooocus/assets/19834515/1efccd91-d4bc-48ff-be80-5d460205092e)

### 1girl, sitting on grass

![image](https://github.com/lllyasviel/Fooocus/assets/19834515/9d17ef84-af00-4751-94df-2117e69e0c06)

### 1girl, fire magic

![image](https://github.com/lllyasviel/Fooocus/assets/19834515/ce8c83d8-4d99-497c-9077-96b9a14577b9)

### 1girl, white wedding dress

![image](https://github.com/lllyasviel/Fooocus/assets/19834515/57f759b3-c9e8-4eeb-810f-4d8edd5f77fc)

### 1girl, armor

![image](https://github.com/lllyasviel/Fooocus/assets/19834515/1e825c04-7927-461b-bab5-d61b3fc19022)

### 1girl, dark magic

![image](https://github.com/lllyasviel/Fooocus/assets/19834515/eaf7216f-7c06-43d8-a870-5bb343402b58)

### 1girl, dragon

![image](https://github.com/lllyasviel/Fooocus/assets/19834515/0fd395b1-567b-4940-8f2d-3c28502d32c1)

**Comment by AdrianNot92** [comment]
Hi, thank you for the great work. Do you have any suggestions on how to fix an issue where my run.bat file is no longer reading information from the config.txt file after the 2.1.864 update?

**Reply by lllyasviel** [reply]
fixed
thanks

**Comment by Phat-Eagle** [comment]
Great! I have a request though. Can you share any prompt tips to get quality results? I am used to those  insane long prompts From A1111.

**Reply by poor7** [reply]
"Insane long prompts" work great in Fooocus too.

**Comment by EgorBragin** [comment]
(To always download new models, use --always-download-new-model )
where do I put it?

**Reply by f0n51** [reply]
Thats an argument for the entry_with_update.py file. Depending on your OS (let's say Windows), just edit your run.bat:
`.\python_embeded\python.exe -s Fooocus\entry_with_update.py entry_with_update.py --always-download-new-model 
pause`

**Reply by EgorBragin** [reply]
thank you very much!

**Reply by JackP07** [reply]
Thx! Small correction:

```
.\python_embeded\python.exe -s Fooocus\entry_with_update.py --always-download-new-model
pause
```

**Reply by NatalyPortman** [reply]
Jack, Where should I insert this into the realistic?
Thanks

**Comment by WerielonOfficial** [comment]
Sir, why everytime gradio.live runs.
it always says "504 bad gateway time out"?? 
i have strong internet connections.
and i always use that fooocus but now i cant use it 😢

**Comment by Gukirt007** [comment]
Hey can anyone suggest how to train this realistic model realisticStockPhoto_v20 with my own photos locally? 
I have hardware access (rtx 4080 16B)🥳

**Comment by matzukowski** [comment]
Hi, can I get more realistic anime style results with v100 or should I look for other model?

**Comment by JohnyMaxida** [comment]
no good ART model for Fooocus?
animaPencil - can draw only Personal art
Not usable for cartoon and pixel art
